### PR TITLE
Update centralized-deployment.md

### PR DESCRIPTION
**Pull request by bhavinkap**
Updated details that iOS and Android support is specific to Outlook. There is no mobile (iOS, Android) story for Word, Excel and PowerPoint

### CODE REVIEW

**Comment by bhavinkap** [comment]
@ElizabethSamuel-MSFT  : Doc update requested

**Comment by o365devx** [comment]
Docs Build status updates of commit _[bd99952](https://github.com/bhavinkap/office-js-docs-pr/commits/bd99952b15286a971276201e2c7fa25bc4d1602d)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/publish/centralized-deployment.md](https://github.com/bhavinkap/office-js-docs-pr/blob/patch-2/docs/publish/centralized-deployment.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/publish/centralized-deployment?branch=pr-en-us-1953) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C6%5C12%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202006120928583239-1953%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=7QPfA25PdGDKbTtalfwYV4QPlAC8%2F2XelpbG6lgFuIk%3D&st=2020-06-12T09%3A25%3A30Z&se=2020-07-13T09%3A30%3A30Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/publish/centralized-deployment.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ The Office 365 admin center makes it easy for an administrator to deploy Office
 The Office 365 admin center currently supports the following scenarios:
 
 - Centralized Deployment of new and updated add-ins to individuals, groups, or an organization.
-- Deployment to multiple platforms, including Windows, Mac, iOS, Android, and on the web.
+- Deployment to multiple platforms, including Windows, Mac and on the web. For Outlook, Deployment to iOS and Android is also supported
```
Comment (ElizabethSamuel-MSFT) [inline docs/publish/centralized-deployment.md:16]:
```suggestion
- Deployment to multiple client platforms, including Windows, Mac, the web, and  iPad. For Outlook, deployment to iOS and Android is also supported.
```

File: docs/publish/centralized-deployment.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ The Office 365 admin center makes it easy for an administrator to deploy Office
 The Office 365 admin center currently supports the following scenarios:
```
Comment (ElizabethSamuel-MSFT) [inline docs/publish/centralized-deployment.md:13]:
```suggestion
The Office 365 admin center currently supports the following scenarios.
```

File: docs/publish/centralized-deployment.md (hunk 1 above)
Comment (bhavinkap) [reply docs/publish/centralized-deployment.md:16]: @ElizabethSamuel-MSFT : 

Support on iPad is Not present for Centralized Deployment.

For Outlook, we only want to call out support on Mobile specifically. Mentioning about iOS, is causing users to assume that iPad is supported as well, but that is not the case. 
CD Support is present only for Outlook on Mobile iOS and Android

While we do not have add-in story for Word, Excel and PowerPoint on Mobile (iOS and Android)

Comment (ElizabethSamuel-MSFT) [reply docs/publish/centralized-deployment.md:16]: Actually, we do have an add-in story for Word, Excel, and PowerPoint on iPad (see the [availability page](https://docs.microsoft.com/en-us/office/dev/add-ins/overview/office-add-in-availability))...

**Comment by o365devx** [comment]
Docs Build status updates of commit _[57f29ff](https://github.com/bhavinkap/office-js-docs-pr/commits/57f29ff156ae7dda3568322d1bc9d8818dcb7806)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/publish/centralized-deployment.md](https://github.com/bhavinkap/office-js-docs-pr/blob/patch-2/docs/publish/centralized-deployment.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/publish/centralized-deployment?branch=pr-en-us-1953) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C6%5C15%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202006151533307625-1953%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=s8PVt9kM544Ac8PV7oXwn%2F6gR7v700r2ITXulVV6ANg%3D&st=2020-06-15T15%3A29%3A45Z&se=2020-07-16T15%3A34%3A45Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Pull request merged**